internal/acctest/config/captchaproviders/captcha_providers_resource_gen_test.go

```diff
@@ -0,0 +1,199 @@
+// Code generated by ping-terraform-plugin-framework-generator
+
+package captchaproviders_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-framework/providerserver"
+	"github.com/hashicorp/terraform-plugin-go/tfprotov6"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/acctest"
+	"github.com/pingidentity/terraform-provider-pingfederate/internal/provider"
+)
+
+const captchaProvidersProvidersId = "captchaProvidersProvidersId"
+
+var testEnvId = ""
+var testRiskPolicyId = ""
```

Comment:
From our discussion, we don't need to use an environment variable for the risk policy id

Reply:
done